src/FixedDataTableCell.js

```diff
@@ -90,15 +82,18 @@ var FixedDataTableCell = createReactClass({
      * Whether touch is enabled or not.
      */
     touchEnabled: PropTypes.bool
-  },
+  };
 
-  getInitialState() {
-    return {
-      isReorderingThisColumn: false,
-      displacement: 0,
-      reorderingDisplacement: 0
-    };
-  },
+  static defaultProps = {
+    align: 'left',
+    highlighted: false,
+  };
+
+  state = {
```

Comment:
We should choose this syntax or setting state on this in the constructor and stick to one.

Reply:
The thing is that if the initial state does not depend on the initial props there is no need for a constructor

Reply:
Ah makes sense, thanks for the explanation